enterprise_access/apps/subsidy_access_policy/migrations/0018_alter_subsidyaccesspolicy_assignment_configuration.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.21 on 2023-11-01 20:35
```

Comment:
This migration is generated with `make migrations` on the `main` branch without any code changes... Should it be committed?

Reply:
I feel like this missing migration removes the field-level unique constraint that was supposed to be removed as part of https://github.com/openedx/enterprise-access/pull/258.  I'd say keeping this migration makes the database constraints more closely match the field definition.

Of course now we also know that mysql doesn't actually support the `unique_together` that was supposed to replace the field-level constraint.  But maybe that's a problem for a separate ticket.

Reply:
Oh, hmm, weird.  FWIW, MySQL does support `unique_together`, it just doesn't support conditional unique constraints.

Reply:
Here's the current UNIQUE keys on the table in our production environment:
```
| subsidy_access_policy_subsidyaccesspolicy | CREATE TABLE `subsidy_access_policy_subsidyaccesspolicy` (
  `created` datetime(6) NOT NULL,
  `modified` datetime(6) NOT NULL,
  `policy_type` varchar(64) NOT NULL,
  `uuid` char(32) NOT NULL,
  ...
  PRIMARY KEY (`uuid`),
  UNIQUE KEY `assignment_configuration_id` (`assignment_configuration_id`),
  UNIQUE KEY `subsidy_access_policy_su_active_assignment_config_f6b904ce_uniq` (`active`,`assignment_configuration_id`),
  ...
```
So the composite key is there, but so is the single-column unique key, which we don't want.  So yes, follow Troy's suggestion and let this migration go in.

Reply:
Sounds good, thanks for the sanity check! Will keep this migration.